### ISSUE TITLE
Can't display blog posts if you set a different home page

### ISSUE THREAD

**Issue by billcreswell** [dignajar/nibbleblog#80]
If you set a different page for a home page, I can't see a way to have a page that lists blog posts.

**Comment by tranberg** [comment]
You are right. However, the posts are still available through the "Latest posts" plugin and they are also visible in your /feed/. I know this is not what you are asking. What would be nice to have is the possibility of an archive page like other blogging platforms have. I have been thinking about this but haven't found the right idea for implementing it on Nibbleblog.

**Comment by billcreswell** [comment]
Yes, having the ability to set a page as the blog page with the list, would move the platform beyond just a blogging platform, and make it a simple website solution.
